docs/reference/commandline/build.md

```diff
@@ -454,20 +454,28 @@ Specifying the `--isolation` flag without a value is the same as setting `--isol
 
 ### <a name="add-host"></a> Add entries to container hosts file (--add-host)
 
-You can add other hosts into a container's `/etc/hosts` file by using one or
-more `--add-host` flags. This example adds a static address for a host named
-`docker`:
+You can add other hosts into a build container's `/etc/hosts` file by using one
+or more `--add-host` flags. This example adds static addresses for hosts named
+`docker` and `dockerv6`:
 
 ```console
-$ docker build --add-host docker:10.180.0.1 .
+$ docker build --add-host docker=10.180.0.1 --add-host dockerv6=2001:db8::33 .
```

Comment:
While we're updating these, perhaps also update the example hostnames (per my other comment)

Reply:
Will do.